packages/components/MenuList/MenuList.js

```diff
@@ -31,7 +31,7 @@ const ColumnHeader = styled.div`
     isDropdownOpened && theme.zIndex.dropdown};
   display: flex;
   align-items: center;
-  height: 56px;
+  min-height: 56px;
```

Comment:
IMHO, it looks bad 
https://scr.4xxi.com/cu/Dashboard_2020-12-17_10_AM-17-35.png
@okomarova need designer help or think up smth other. maybe ellipsis?

Reply:
And, sorry, I'm using your styles, and now dropdown is broken
https://scr.4xxi.com/cu/Dashboard_2020-12-17_10_AM-21-11.png